docs/prow/developers-faq.md

```diff
@@ -0,0 +1,51 @@
+# Developers FAQ
+Frequently Asked Questions (FAQ) a developer can have, regarding working with Prow as a Continuous Integration (CI) mechanism.
+
+**Q: I would like to add a new component/item and have it operated by Prow**
+
+**A:** Creating a CI pipeline for a new component requires a new ProwJob. Please see [this document](create-component-jobs.md) for further information
```

Comment:
In my opinion, we should combine first 2 questions in one answer, if you want to add a new component, you need to create jobs for the master branch, as well as a release branch.